sys/include/net/gnrc/lwmac/lwmac.h

```diff
@@ -216,54 +243,59 @@ extern "C" {
  * LWMAC supports burst transmission based on the pending-bit technique, and this macro
  * here defines the largest number of packets allowed to be sent in one consecutive
  * sequence. In case a sender has multi packets for one receiver,the burst transmission
- * procedure is as follow: 1) the sender first uses WR stream to locate the receiver's
- * wake-up period (if the sender has already phase-locked the receiver's phase, normally
- * the sender only cost one WR to get the first WA from the receiver) and then sends its
- * first data. 2) After the transmission of the first data, the sender immediately sends
- * a WR to the receiver for starting the second round of transmission of the second data.
- * The receiver should also immediately reply WA for continue receiving data packets. In
- * case the sender doesn't receive WA during LWMAC_TIME_BETWEEN_WR_US, it regards the
+ * procedure is as follow:
+ * (1) the sender first uses WR stream to locate the receiver's wake-up period (if the
```

Comment:
Please use markdown lists, so doxygen parses this properly:

```
procedure is as follows:

1. ....
```

(also keep in mind to indent after a new line).